frontend/src/components/GameConsole/GameConsole.js

```diff
@@ -54,8 +54,7 @@ class GameConsole extends React.Component {
 
         <form className="submit_game_command"
           onSubmit={this.SubmitGameCommand}
-          onChange={this.handleChange}
-          className="submit_game_command">
```

Comment:
className is repeated twice (Line 55 and 58)

Reply:
removed extra <form ... className